### PR TITLE
Enable Multiple Reports

### PR DESCRIPTION
**Pull request by jasondegraw**
This PR enables multiple reports in a single BuildingSync file. Presently, only one is allowed and it is not possible to associate the report with just a part of the building or with a particular system. These changes will make those things possible, but the changes are breaking.

### CODE REVIEW

File: proposals/2019/Enable Multiple Reports.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## Justification
+A single report per file limits the extent to which reports can be associated with different parts of buildings or systems. This is particularly an issue when data has been collected for part of a building, but the report implicitly applies to all buildings in the file. Furthermore, BuildingSync should be able to track changes over time, and allowing multiple reports means that this will be easier to do. Users can simply add a new report if another audit is done rather than add to previous data. While this isn't strictly necessary, it is a slightly more natural representation and should be easier on users.
+
+## Implementation
+Four steps will be taken to make this change:
+
+  1. The report element will be changed into a containerized list: `<Reports><Report/></Reports>`.
+  2. An ID attribute will be added to the `Report` type. This attribute will not be used elsewhere in the schema yet, but it is good practice to have identifiers available.
```
Comment (nllong) [inline proposals/2019/Enable Multiple Reports.md]: This has been done in this PR https://github.com/BuildingSync/schema/pull/80

File: proposals/2019/Enable Multiple Reports.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  3. Building off of the `LinkedPremises` type, create a linkage type that allows users to specify that the report applies to a premises or a system. It is possible that a union type will accomplish this.
+  4. Modify the example files to use the containerized lists of reporting.
+
+Steps 2 and 3 are not breaking changes, but step 1 is. If we are rigidly following semantic versioning, at the completion of this modification the version number of the schema will need to be bumped to 2.0.
```
Comment (nllong) [inline proposals/2019/Enable Multiple Reports.md]: Yes, I think we are moving to 2.0 on July 5.

**Comment by nllong** [comment]
Does this address #74?

File: proposals/2019/Enable Multiple Reports.md (hunk 1 above)
Comment (jasondegraw) [reply proposals/2019/Enable Multiple Reports.md]: Oops, forgot about that. Will edit this a bit.

**Comment by jasondegraw** [comment]
@nllong Yes, it will address #74, at least partly. I feel like there needs to be an example file that does something like that more explicitly. There may not be time right now, so maybe put in another issue to keep track of the need?

**Comment by jasondegraw** [comment]
@nllong Fixed up the merge issue, I think this is ready to merge now.

**Comment by jasondegraw** [comment]
Once CI goes green, will merge.

**Pull request merged**